exercises/nucleotide-count/example.R

```diff
@@ -0,0 +1,17 @@
+library(magrittr)
```

Comment:
I like this pkg very much and using the  ["standard library" is only mentioned with regards to the tests](https://github.com/exercism/docs/blob/master/maintaining-a-track/reviewing-a-pull-request.md#encourage-best-practices). I think because piping is becoming more popular in the R community, we should start including it in examples. Maybe not re-write old examples, though ;-)